docs/guide/content_rendering.md

```diff
@@ -1073,31 +1174,30 @@ In this example, as the `ViewController` is not applied, the custom controller t
 
 namespace Acme\TestBundle\Controller;
 
-use Symfony\Component\HttpFoundation\Response;
 use eZ\Bundle\EzPublishCoreBundle\Controller;
+use eZ\Publish\Core\MVC\Symfony\View\ContentView;
 
 class DefaultController extends Controller
 {
-    public function viewFolderAction( $locationId, $layout = false, $params = array() )
+    public function viewFolderAction(ContentView $view)
```

Comment:
@bdunogier Is this example correct if we use ContentView?
If so some of the default logic is used so it might be a bit waste or?

If not, I guess we should expand intro here to make it clear that the difference here is that code takes care about rendering and the response itself.

Reply:
Yes, this example is correct with `ContentView`